presto-main/src/main/java/io/prestosql/testing/LocalQueryRunner.java

```diff
@@ -361,7 +361,7 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
                 new EventListenerManager(),
                 new SessionPropertyDefaults(nodeInfo));
 
-        connectorManager.addConnectorFactory(globalSystemConnectorFactory);
+        connectorManager.addConnectorFactory(globalSystemConnectorFactory, () -> getClass().getClassLoader());
```

Comment:
Same comment as `TestingPrestoServer`, probably want to use the class loader of the instances. Not that it actually matters...